DataDefinitions/Properties/Commodities.resx

```diff
@@ -1493,4 +1493,8 @@
     <value>Torval Commercial Contracts</value>
     <comment>Commodity torvalcommercialcontracts</comment>
   </data>
+  <data name="Unknown" xml:space="preserve">
+    <value>Unknown </value>
+    <comment>Commodity Unknown</comment>
+  </data>
```

Comment:
As above. Is there a more descriptive name we can use? `Unknown mission cargo` or similar?

Reply:
My intent was to create a generic description that could be used in multiple contexts. 'Unknown mission cargo' would be too specific, IMO.

Reply:
Also I don't think it should have a trailing space.